source/Calamari.Common/Plumbing/Variables/TentacleVariables.cs

```diff
@@ -14,6 +14,7 @@ public static class Agent
             public static readonly string InstanceName = "Octopus.Tentacle.Agent.InstanceName";
             public static readonly string ApplicationDirectoryPath = "Octopus.Tentacle.Agent.ApplicationDirectoryPath";
             public static readonly string JournalPath = "env:TentacleJournal";
+            public static readonly string TentacleHome = "env:TentacleHome";
```

Comment:
The existing usages of `TentacleHome` do something like `Environment.GetEnvironmentVariable("TentacleHome")`. But since we have the variables bag I think it makes sense to just grab it from there in the same way we're trying to get the journal path variable.